.circleci/cimodel/data/windows_build_definitions.py

```diff
@@ -0,0 +1,137 @@
+import cimodel.lib.miniutils as miniutils
+
+
+NON_PR_BRANCH_LIST = [
+    "master",
+    r"/ci-all\/.*/",
+    r"/release\/.*/",
+]
+
+
+class WindowJob:
+    def __init__(self,
+                 test_index,
+                 vscode_spec,
+                 cuda_version,
+                 force_on_cpu=False):
+
+        self.test_index = test_index
+        self.vscode_spec = vscode_spec
+        self.cuda_version = cuda_version
+        self.force_on_cpu = force_on_cpu
+        self.run_on_prs = vscode_spec.year != 2019
```

Comment:
Can it be a `__init__` argument, for example `run_on_prs_pred = lambda job: job.vscode_spec.yer != 2019` ?

Reply:
sure